### PR TITLE
Update virtual.py

### PR DESCRIPTION
**Pull request by santosh-sudo**
I forgot to print the speak function under speech recogniser .

### CODE REVIEW

File: virtual.py [hunk 1]
```diff
@@ -45,4 +45,8 @@ def takeCommand():
             return "None"
         return statement
     
+speak("Loading your AI personal assistant G-One")
+wishMe()
+
```
Comment (shivambitm) [inline virtual.py:50]: okay no problem I'm merging this also.

**Review by santosh-sudo — COMMENTED** [review]
yes merge it

File: virtual.py (hunk 1 above)
Comment (ayushii18) [inline virtual.py:50]: nice , could be added later also

**Review by ayushii18 — APPROVED** [review]
nice

File: virtual.py (hunk 1 above)
Comment (shivambitm) [reply virtual.py:50]: thanks for your review.

**Review by shivambitm — APPROVED** [review]

**Review by ayushii18 — APPROVED** [review]

**Pull request merged**